assets/styles/variables/_specials.scss

```diff
@@ -428,6 +428,11 @@ $sidebar-font-weight: normal !default;
 /// @type String
 $sidebar-align: left !default;
 
+/// Line Height for Sidebar on `<p>` elements that are children of elements with a class of `sidebar`.
+/// @type String 
+/// /// @since 1.1.0
+$sidebar-line-height: 1.4em !default;
```

Comment:
@dannylonglegs Was this inherited from somewhere else before? I'm concerned this might be a breaking change.